ci_build/outer_launch_tests.sh

```diff
@@ -8,5 +8,6 @@ STACK_RESOLVER=${STACK_RESOLVER:-lts-6.2}
 IMAGE_NAME=tensorflow/haskell/ci_build:$STACK_RESOLVER
 
 git submodule update
+docker pull tensorflow/tensorflow:nightly-devel
```

Comment:
Not necessary in this PR, but: what do you think about making the dockerfiles more specific than "nightly" so they correspond more stably to the submodule's version?  For example, picking a release like `tensorflow/tensorflow:0.12.1-devel`:
https://hub.docker.com/r/tensorflow/tensorflow/tags/